source/js/components/petition/petition-thank-you.jsx

```diff
@@ -0,0 +1,180 @@
+import { Component } from "react";
+import { ReactGA } from "../../common";
+import DonationModal from "./donation-modal.jsx";
+import copyToClipboard from "../../copy-to-clipboard";
+
+/**
+ * Thank you screen for FormAssembly petitions
+ * This is a simplified version of the original Petition component
+ */
+class PetitionThankYou extends Component {
+  constructor(props) {
+    super(props);
+    this.state = this.getInitialState();
+
+    // Do we have modal data?
+    this.modals = false;
+    if (this.props.modals) {
+      this.modals = this.props.modals;
+      try {
+        this.modals = JSON.parse(this.modals);
+      } catch (e) {
+        this.modals = false;
+        console.error(`Could not parse modal data from petition markup.`);
+      }
+    }
+  }
+
+  getInitialState() {
+    return {
+      showDonationModal: true,
+    };
+  }
+
+  // [FA TODO] set up GA event tracking for form submissions
+  setAnalytics() {
+    // window.dataLayer = window.dataLayer || [];
```

Comment:
This came from `petition.jsx`. I commented out analytics related code for now. We'll get to these once we have a plan for what we wanna track (https://github.com/MozillaFoundation/foundation.mozilla.org/issues/10470)